src/MotionControl.h

```diff
@@ -8,7 +8,7 @@
 #ifndef MOTOROS2_MOTION_CONTROL_H
 #define MOTOROS2_MOTION_CONTROL_H
 
-#define START_MAX_PULSE_DEVIATION           30
+#define START_MAX_PULSE_DEVIATION           10
```

Comment:
I'm not sure this would work -- as @ted-miller wrote in #233, we might need a separate value for detecting the end of motion. There have been users who've had to increase this value (for larger / heavier robots fi) which would then break the end-of-motion detection.

Reply:
Would we want to remove this from the PR and make a separate issue for dealing with detecting the end of motion?

Reply:
I'd suggest so.

This value actually already makes it impossible for my robot here (GP25) to start any motion.